<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Список комманд:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_pauseOffset">
        <property name="text">
         <string>Отступ после паузы в мс:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkBox_copyPhraseTextAC">
        <property name="text">
         <string>Копировать текст при деление фразы</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="checkBox_savePhraseText">
        <property name="text">
         <string>Сохранять текст во время редактирования</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Скорость воспроизведения (множитель):</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_playSpeed">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBox_pauseOffset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox_playAfterChangePhrase">
        <property name="text">
         <string>Воспроизводить при смене фраз</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="checkBox_autoScalePlot">
        <property name="text">
         <string>Автоматически масштабировать график при смене фразы</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="checkBox_autoMoveMarker">
        <property name="text">
         <string>Автоматически переводить маркер в положение бегунка после остановки</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="checkBox_playPosTracking">
        <property name="text">
         <string>Сдвигать осцилограмму при воспроизведении</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidget_actionList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Список комманд&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Комманды</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Горячие клавиши</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>фыв</string>
         </property>
        </item>
        <item row="12" column="1">
         <property name="text">
          <string>йцу</string>
         </property>
        </item>
        <item row="13" column="1">
         <property name="text">
          <string>кйцкуйцу</string>
         </property>
        </item>
        <item row="14" column="0">
         <property name="text">
          <string>уйцу</string>
         </property>
        </item>
        <item row="14" column="1">
         <property name="text">
          <string>йцуйцу</string>
         </property>
        </item>
        <item row="15" column="0">
         <property name="text">
          <string>йц</string>
         </property>
        </item>
        <item row="15" column="1">
         <property name="text">
          <string>й</string>
         </property>
        </item>
        <item row="16" column="1">
         <property name="text">
          <string>цуйцуйцу</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QCheckBox" name="checkBox_checkPhrasesDuraton">
        <property name="text">
         <string>Проверять общую длительность фраз с заданным именем, после удаления (мс)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="lineEdit_phraseName"/>
      </item>
      <item row="13" column="2">
       <widget class="QSpinBox" name="spinBox_phraseDurationThreshold">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="checkBox_editPhraseNameInMarking">
        <property name="text">
         <string>Редактировать текст фразы в разметке</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QCheckBox" name="checkBox_showPluginMassages">
        <property name="text">
         <string>Автоматически отображать окно сообщений плагинов.</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QCheckBox" name="checkBox_openMarkingAsWave">
        <property name="text">
         <string>Открывать одноимённый файл разметки и описания сигнала</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QCheckBox" name="checkBox_openMarkingAsUTF8">
        <property name="text">
         <string>Открывать/сохранять файлы разметки как UTF-8</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Шрифт программы:</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Шрифт разметки:</string>
        </property>
       </widget>
      </item>
      <item row="20" column="2">
       <widget class="QPushButton" name="pushButton_setFont">
        <property name="text">
         <string>Настроить</string>
        </property>
       </widget>
      </item>
      <item row="21" column="2">
       <widget class="QPushButton" name="pushButton_setMarkingFont">
        <property name="text">
         <string>Настроить</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QCheckBox" name="checkBox_autoSaveMarking">
        <property name="text">
         <string>Автоматически сохранять разметку после редактирования</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QCheckBox" name="checkBox_autoSaveSignalDesc">
        <property name="text">
         <string>Автоматически сохранять описание сигнала после редактирования</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QCheckBox" name="checkBox_show_signal_desc_in_startup">
        <property name="text">
         <string>Отображать описание сигнала при старте</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_pauseOffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_pauseOffset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_checkPhrasesDuraton</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_phraseName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_checkPhrasesDuraton</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_phraseDurationThreshold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
